LDM-562.tex

```diff
@@ -42,6 +47,993 @@
 \begin{document}
 \maketitle
 
+\section{\citeds{LSE-61} Flowdown}
+
+This section contains direct copies of requirements from \citeds{LSE-61}, based on the Level 1 requirements defined in \citeds{LDM-148}.
```

Comment:
LDM-148 doesn't define requirements, it simply maps requirements to components.

I think it would be super helpful to list in this document the LDM-148 components which it covers.

Also note that this is the L2 requirements document. :-)